### PR TITLE
Tick class fix

### PR DESCRIPTION
**Pull request by timAmida**
# Related Tickets

# Other Repos' PR(s) Intended to Work With This PR
-None

# How Things Worked (or Didn't) Before This PR
-Classnames changed Tickchange function stopped working.
# How Things Work Now (And How to Test)
- Classname in TickChange function fixed.
# Readiness

1. [x] This PR passes all automated tests.
2. [x] This PR has no linting errors.
3. [x] This PR's changes to configuration files have been documented in all appropriate places (such as but not limited to README.md), if applicable.

### CODE REVIEW

File: src/components/ChartContainer/ChartBarDateRange.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
-    );
-}
-
-export default ChartBarDateRange;
+import React from 'react';
+
+function ChartBarDateRange() {
+  return (
+    <div>
+      DataRange
```
Comment (jmfadeley) [inline src/components/ChartContainer/ChartBarDateRange.js:6]: Data range or date range?

File: src/components/ChartContainer/ChartBarDateRange.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
-function ChartBarDateRange() {
-    return (
-        
-    );
-}
-
-export default ChartBarDateRange;
+import React from 'react';
+
+function ChartBarDateRange() {
```
Comment (anthem123) [inline src/components/ChartContainer/ChartBarDateRange.js:3]: It looks like this class was changed by mistake. If so it should be reverted.

Also does anyone know what this class is even doing?

Comment (jmfadeley) [inline src/components/ChartContainer/ChartBarDateRange.js:3]: I don't see the purpose or use of this class anywhere.

Comment (timAmida) [reply src/components/ChartContainer/ChartBarDateRange.js:3]: delete it

**Pull request merged**